modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/SwiftCodegen.java

```diff
@@ -337,7 +337,7 @@ public CodegenProperty fromProperty(String name, Property p) {
             // Swift compiler will generate an error
             if (isReservedWord(codegenProperty.datatypeWithEnum) ||
                     name.equals(codegenProperty.datatypeWithEnum)) {
-                codegenProperty.datatypeWithEnum = escapeReservedWord(codegenProperty.datatypeWithEnum);
+                codegenProperty.datatypeWithEnum = codegenProperty.datatypeWithEnum + "Enum";
                     }
```

Comment:
do you mind indenting the line with 4 additional spaces to make it align with the previous line?